File: Python-Basics/loops/coins.py
change = float(input())
coins = round(change * 100)
coin_count = 0

while(coins > 0):

    if coins >= 200:
        coins -= 200
        coin_count += 1
    elif coins >= 100:
        coins -= 100
        coin_count += 1
    elif coins >= 50:
        coins -= 50
        coin_count += 1
    elif coins >= 20:
        coins -= 20
        coin_count += 1
    elif coins >= 20:
        coins -= 20
        coin_count += 1
    elif coins >= 10:
        coins -= 10
        coin_count += 1
    elif coins >= 5:
        coins -= 5
        coin_count += 1
    elif coins >= 2:
        coins -= 2
        coin_count += 1
    elif coins >= 1:
        coins -= 1
        coin_count += 1

print(coin_count)

File: CPP-Basics/Coins.cpp
#include <iostream>

using namespace std;

int main () {

    double change;
    cin >> change;

    int coins = change * 100;

    int coinCount = 0;

    while(coins > 0) {

        ///2, 1, 50, 20, 10, 5, 2, 1

        if(coins >= 200){

            coins -= 200;
            coinCount ++;

        } else if (coins >= 100){

            coins -= 100;
            coinCount ++;

        } else if (coins >= 50){

            coins -= 50;
            coinCount ++;

        } else if (coins >= 20){

            coins -= 20;
            coinCount ++;

        } else if (coins >= 10){

            coins -= 10;
            coinCount ++;

        } else if (coins >= 5){

            coins -= 5;
            coinCount ++;

        } else if (coins >= 2){

            coins -= 2;
            coinCount ++;

        } else if (coins >= 1){

            coins -= 50;
            coinCount ++;

        }
    }

    cout << coinCount << endl;

    return 0;
}
